pages/account/manage/testimonials.js

```diff
@@ -108,78 +108,75 @@ export default function ManageTestimonials({ BASE_URL, testimonials }) {
           </h3>
         </div>
 
-        {!testimonialList?.length && (
+        {!testimonialList?.length ? 
+         <div>
+         <div className="flex gap-4">
```

Comment:
The indentation doesn't look quite right here? Try using the Prettier plugin for VScode, it should fix on save